curriculum/challenges/english/08-coding-interview-prep/rosetta-code/largest-int-from-concatenated-ints.md

```diff
@@ -0,0 +1,74 @@
+---
+id: 5a23c84252665b21eecc7edb
+title: Largest int from concatenated ints
+challengeType: 5
+---
+
+## Description
+<section id='description'>
+Given a set of positive integers, write a function to order the integers in such a way that the concatenation of the numbers forms the largest possible integer and return this integer.
```

Comment:
This is the instruction, move it to there please

Reply:
I not sure this is necessary. I came across a lot of challenges like this where the description was really short. When I left the description empty and moved the sentence or two to the instructions section there was an extra `<hr>` above the instructions. I'm thinking it's better just to leave shorter descriptions like this alone.